torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -598,7 +598,7 @@ def _add_cutlass_gemm_choices(
             input_strides = [node.get_stride() for node in input_nodes]
             output_layout = layout
             warning_msg = f"No suitable Cutlass GEMM configs found, fallbacks used ( {len(ops)=}, {output_layout=}, {input_layouts=}, {input_strides=} )"  # noqa: B950
```

Comment:
Nit: this warning message is constructed whether it's needed to for logging. Use Loggers builtin string interpolation with lazy %s formatting. We also have ruff statically verify it.